__tests__/app.test.js

```diff
@@ -0,0 +1,41 @@
+const request = require('supertest')
+const app = require('../app')
+const db = require('../db/connection.js')
+const seed = require('../db/seeds/seed')
+const testData = require('../db/data/test-data')
+
+beforeEach(() => seed(testData))
+afterAll(() => db.end())
+
+
+describe("app", () => {
+    describe("GET /api/topics", () => {
+        test("should return an array of objects with properties of slug and description", () => {
+            return request(app).get('/api/topics').expect(200).then((res) => {
+                expect(res.body.topics[0]).toMatchObject({
```

Comment:
It may also be worth adding a test to check the length of the array. To ensure we act getting back the data we expect :)